.circleci/config.yml

```diff
@@ -449,13 +449,13 @@ jobs:
           message: 'VSCode Extensions v${RELEASE_VERSION} have been published to the marketplace'
       - slack/notify:
           channel: 'pdt_releases'
-          color: '#9bcd9b'
-          message: 'VSCode Extensions v${RELEASE_VERSION} Post Publish - Merging main into develop'
+          color: '#FFDEAD'
```

Comment:
You of all people Shelby, I can't believe you'd declare final fantasy dead.

Reply:
🤣  oh that is perfect